sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Main.java

```diff
@@ -4,13 +4,15 @@
 package com.azure.cosmos.benchmark;
 
 import com.azure.cosmos.benchmark.ctl.AsyncCtlWorkload;
+import com.azure.cosmos.benchmark.linkedin.CtlWorkload;
 import com.beust.jcommander.JCommander;
 import com.beust.jcommander.ParameterException;
+import java.util.Optional;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import static com.azure.cosmos.benchmark.Configuration.Operation.CtlWorkload;
-import static com.azure.cosmos.benchmark.Configuration.Operation.ReadThroughputWithMultipleClients;
+import static com.azure.cosmos.benchmark.Configuration.Operation.*;
```

Comment:
as code style we don't use wildcard import.

Please change your IDE default formatting:
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports